app/controllers/healthchecks_controller.rb

```diff
@@ -1,5 +1,5 @@
 class HealthchecksController < ApplicationController
-  skip_before_action :site_wide_authentication
+  skip_before_action :http_basic_authenticate
```

Comment:
Instead of skipping the action we should maybe return `false` from the `authenticate?` method here?